src/main/java/org/opensearch/security/tools/SecurityAdmin.java

```diff
@@ -577,39 +577,44 @@ public static int execute(final String[] args) throws Exception {
             )
         ) {
 
-            Response whoAmIRes = restHighLevelClient.getLowLevelClient().performRequest(new Request("GET", "/_plugins/_security/whoami"));
-            if (whoAmIRes.getStatusLine().getStatusCode() != 200) {
-                System.out.println("Unable to check whether cluster is sane because return code was " + whoAmIRes.getStatusLine());
-                return (-1);
-            }
+            if (whoami) {
+                Response whoAmIRes = restHighLevelClient.getLowLevelClient()
+                    .performRequest(new Request("GET", "/_plugins/_security/whoami"));
+                if (whoAmIRes.getStatusLine().getStatusCode() != 200) {
+                    System.out.println("Unable to check whether cluster is sane because return code was " + whoAmIRes.getStatusLine());
```

Comment:
```suggestion
                    System.out.println("Unable to check whether cluster is successfully initialized because return code was " + whoAmIRes.getStatusLine());
```

Sane may have connotations so we may want to use another phrase.